app/src/main/java/io/homeassistant/companion/android/controls/HaControlsProviderService.kt

```diff
@@ -61,6 +62,7 @@ class HaControlsProviderService : ControlsProviderService() {
         "media_player" to null,
         "remote" to null,
         "scene" to SceneControl,
+        "script" to SceneControl,
```

Comment:
If a script has a `delay` then technically I think it can be stopped by turning it off.  Unless that has changed recently?

Reply:
Oh I didn't know that! Although I think it would be kinda unintuitive in most cases if the scripts state changes from Off to On and instantly back to Off

Reply:
Looks like the frontend ditched showing a toggle when it needs to and just shows `execute` now so I think it should be ok :)